packages/sdks/typescript/tester-local/__tests__/test_search_patient.ts

```diff
@@ -0,0 +1,73 @@
+import * as dotenv from "dotenv";
+import * as path from "path";
+dotenv.config({ path: path.resolve(__dirname, '.env') });
+
+import { MetriportClient } from "../src";
+import { BasePatient, Demographics } from "../src/api/resources/medical/resources/patient/types";
+import { PatientCreate } from "../src/api/resources/medical/resources/patient/client/requests";
+import { UsState, Address } from "../src/api/resources/commons/types";
+
+const apiUrl = process.env.BASE_URL;
+const apiToken = process.env.API_KEY;
+const facilityId = process.env.FACILITY_ID;
+
+if (!apiUrl || !apiToken || !facilityId) {
+  throw new Error("Required environment variables are not set");
+}
+
+const client = new MetriportClient({
```

Comment:
Defs NABD, but I suggest naming `client` as `metriport` since this can be used by others (copy/paste) to build their own code, this "encourages" them to name our SDK w/ our name. :) Likewise to other SDKs/tests if we take this road.